src/renderer/services/wallet/balances.ts

```diff
@@ -75,31 +76,31 @@ export const createBalancesService = ({
   // reload all balances
   const reloadBalances: FP.Lazy<void> = () => {
     userChains$.pipe(RxOp.take(1)).subscribe((enabledChains) => {
-      if (enabledChains.includes(BTCChain)) BTC.reloadBalances()
-      if (enabledChains.includes(DASHChain)) DASH.reloadBalances()
-      if (enabledChains.includes(BCHChain)) BCH.reloadBalances()
-      if (enabledChains.includes(ETHChain)) ETH.reloadBalances()
-      if (enabledChains.includes(ARBChain)) ARB.reloadBalances()
-      if (enabledChains.includes(AVAXChain)) AVAX.reloadBalances()
-      if (enabledChains.includes(BASEChain)) BASE.reloadBalances()
-      if (enabledChains.includes(BSCChain)) BSC.reloadBalances()
-      if (enabledChains.includes(THORChain)) THOR.reloadBalances()
+      if (enabledChains.includes(BTCChain)) BTC.reloadBalances(DEFAULT_WALLET_TYPE)
```

Comment:
How about giving a default option instead of passing it?

```
const reloadBalances  = (walletType: WalletType = DEFAULT_WALLET_TYPE) => {
    ...
}
```